identify/extensions.py

```diff
@@ -120,10 +120,13 @@
     'lean': {'text', 'lean'},
     'lektorproject': {'text', 'ini', 'lektorproject'},
     'less': {'text', 'less'},
+    'lfm': {'text', 'lazarus', 'form'},
```

Comment:
should this be `lazarus-form` ?

Reply:
I think we need to preserve the `lazarus` tag, as `.lfm` is a Lazarus specific file, but I changed `form` to `lazarus-form`.